mne/epochs.py

```diff
@@ -818,6 +869,59 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         return self._evoked_from_epoch_data(data, self.info, picks, n_events,
                                             kind, self._name)
 
+    def regress(self, on, fit_intercept=True, by=None):
```

Comment:
+1 for leaving it out for now. I agree with @kingjr that it's better to keep it separate if possible. We can always add it later if we need to, let's proceed in multiple PRs to simplify API considerations.